### PR TITLE
RHDEVDOCS-6052: Content creation for GitOps 1.12.3 RN

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.12++

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6052

Link to docs preview:

Release Notes change: https://75959--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes.html#gitops-release-notes-1-12-3_gitops-release-notes

Argo CD CLI change: https://75959--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes.html#GitOps-compatibility-support-matrix_gitops-release-notes

SME review: @reginapizza 
QE review: @varshab1210 
Internal peer review: @eromanova97 
Peer review: @tmalove 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/gitops-release-notes-1-12-3.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+{gitops-title} 1.12.3 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12.3_{context}"]
+== Errata updates
+
+[id="RHSA-XXXX:NNNN-gitops-1-12-3-security-update-advisory_{context}"]
+=== RRHSA-2024:131901-04 - {gitops-title} 1.12.3 security update advisory
+
+Issued: 2024-05-15
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-12-3.adoc:18]: Please change the date to 23rd for now

**Review by varshab1210 — COMMENTED** [review]
Left a minor comment, LGTM otherwise

File: modules/gitops-release-notes-1-12-3.adoc (hunk 1 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-3.adoc]: Fixed.

File: modules/gitops-release-notes-1-12-3.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+{gitops-title} 1.12.3 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12.3_{context}"]
+== Errata updates
+
+[id="rhsa-2024:3368-04-gitops-1-12-3-security-update-advisory_{context}"]
+=== RHSA-2024:3368-04 - {gitops-title} 1.12.3 security update advisory
+
+Issued: 2024-05-23
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-12-3.adoc:18]:
```suggestion
Issued: 2024-05-28
```

File: modules/gitops-release-notes-1-12-3.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:3368-04[RHSA-2024:3368-04]
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-12-3.adoc:22]: I am not able to access the link

```suggestion
* link:https://access.redhat.com/errata/RHSA-2024:3368[RHSA-2024:3368-04]
```

Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-3.adoc:22]: Fixed.

File: modules/gitops-release-notes-1-12-3.adoc [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-12-3_{context}"]
+== Fixed issues
+
+* Before this update, pods in a different namespace could access the Redis server on port 6379 to obtain read and write access to the data. This issue has been fixed in this release by enabling secure authentication.
```
Comment (eromanova97) [inline modules/gitops-release-notes-1-12-3.adoc:34]:
```suggestion
* Before this update, pods in a different namespace could access the Redis server on port `6379` to obtain read and write access to the data. This update fixes the issue by enabling secure authentication.
```

File: modules/go-compatibility-and-support-matrix.adoc [hunk 5]
```diff
@@ -21,15 +21,15 @@ In {OCP} 4.13, the `stable` channel has been removed. Before upgrading to {OCP}
 
 [options="header"]
 |===
-|OpenShift GitOps 7+|Component Versions|OpenShift Versions
+|OpenShift GitOps 8+|Component Versions|OpenShift Versions
```
Comment (eromanova97) [inline modules/go-compatibility-and-support-matrix.adoc:24]: One comment about the format of this table. Based on our RH guidelines: https://redhat-documentation.github.io/supplementary-style-guide/#accessibility-tables, we should avoid tables with irregular headers; and it links us to this example: https://www.w3.org/WAI/tutorials/tables/irregular/#table-with-two-tier-headers

So I think you should probably create an issue for this and fix it (but definitely not as part of this RN PR)

cc: @Srivaralakshmi

**Review by eromanova97 — COMMENTED** [review]
Just one suggetsion comment and one additional comment regarding the formatting of your matrix table. The rest looks good, thank you!

File: modules/gitops-release-notes-1-12-3.adoc [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+[id="rhsa-2024:3368-04-gitops-1-12-3-security-update-advisory_{context}"]
+=== RHSA-2024:3368-04 - {gitops-title} 1.12.3 security update advisory
+
+Issued: 2024-05-28
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:3368[RHSA-2024:3368-04]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
```
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-12-3.adoc:24]: What is this change about ?

**Review by iam-veeramalla — APPROVED** [review]

File: modules/gitops-release-notes-1-12-3.adoc (hunk 6 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-3.adoc:24]: This is the normal text that's used for all the errata releases @iam-veeramalla. Adding a reference here:

https://docs.openshift.com/gitops/1.10/release_notes/gitops-release-notes.html#errata-updates-1-10-4_gitops-release-notes

**Comment by iam-veeramalla** [comment]
LGTM

File: modules/gitops-release-notes-1-12-3.adoc (hunk 6 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-3.adoc:24]: Fixed.

**Comment by gaurav-nelson** [comment]
@Dhruv-Soni11 Can you please rebase this branch and put for merge review again?

**Comment by Dhruv-Soni11** [comment]
> @Dhruv-Soni11 Can you please rebase this branch and put for merge review again?

Hi @gaurav-nelson - I have gone ahead and rebased the PR. PTAL. Thanks.

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 LGTM

**Comment by Srivaralakshmi** [comment]
#
/cherrypick gitops-docs-1.12

**Pull request merged**